lua/focus/modules/resizer.lua

```diff
@@ -21,76 +21,180 @@ local golden_ratio_minheight = function()
     return math.floor(golden_ratio_height() / (3 * golden_ratio))
 end
 
+local easing = {}
+
+function easing.linear(t, b, c, d)
+    return c * t / d + b
+end
+
```

Comment:
These won't all be here, just testing what looks best and I'm planning on choosing 3 or 4 to have as options.